src/integrations/settings-integration.php

```diff
@@ -481,6 +484,82 @@ protected function get_site_represents_person( $settings ) {
 		return $person;
 	}
 
+	/**
+	 * Get site policy data.
+	 *
+	 * @param array $settings The settings.
+	 *
+	 * @return array The policy data.
+	 */
+	private function get_site_basics_policies( $settings ) {
+		$policies = [];
+
+
+		$policies = $this->maybe_add_policy( $policies, $settings['wpseo_titles']['publishing_principles_id'], 'publishing_principles_id' );
+		$policies = $this->maybe_add_policy( $policies, $settings['wpseo_titles']['ownership_funding_info_id'], 'ownership_funding_info_id' );
+		$policies = $this->maybe_add_policy( $policies, $settings['wpseo_titles']['actionable_feedback_policy_id'], 'actionable_feedback_policy_id' );
+		$policies = $this->maybe_add_policy( $policies, $settings['wpseo_titles']['corrections_policy_id'], 'corrections_policy_id' );
+		$policies = $this->maybe_add_policy( $policies, $settings['wpseo_titles']['ethics_policy_id'], 'ethics_policy_id' );
+		$policies = $this->maybe_add_policy( $policies, $settings['wpseo_titles']['diversity_policy_id'], 'diversity_policy_id' );
+		$policies = $this->maybe_add_policy( $policies, $settings['wpseo_titles']['diversity_staffing_report_id'], 'diversity_staffing_report_id' );
+
+		$policies = $this->append_newest_posts( $policies );
+
+		return $policies;
+	}
+
+	/**
+	 * This adds 10 pages to the policy options.
+	 *
+	 * @param array $policies The current list of policy options.
+	 *
+	 * @return mixed
+	 */
+	private function append_newest_posts( $policies ) {
+		$posts = get_pages(
+			[
+				'post_type'   => 'page',
+				'post_status' => 'publish',
+				'number'      => 10,
+			]
+		);
+
+		foreach ( $posts as $post ) {
+			$formatted_post         = [];
+			$formatted_post['id']   = $post->ID;
+			$formatted_post['name'] = $post->post_title;
+			$policies[]             = $formatted_post;
+		}
+
+		return $policies;
+	}
+
+	/**
+	 * Adds policy data if it is present.
+	 *
+	 * @param array  $policies The existing policy data.
+	 * @param string $policy   The policy id to check.
+	 * @param string $key      The option key name.
+	 *
+	 * @return array The policy data.
+	 */
+	private function maybe_add_policy( $policies, $policy, $key ) {
+		$policy_array = [
+			'id'   => false,
+			'name' => '',
+		];
+		if ( isset( $policy ) ) {
```

Comment:
I would change the `isset` to something more specific like `is_int()`. 

According to the [get_post()](https://developer.wordpress.org/reference/functions/get_post/) docs:
> null, false, 0 and other PHP falsey values return the current global post inside the loop

Reply:
Removed this code :)